lib/worker/src/gcs.rs

```diff
@@ -39,6 +43,34 @@ impl GcsClient {
         Ok(Self {
             inner: Client::new(gcs_config),
             bucket,
+            endpoint: endpoint.unwrap_or_else(|| "https://localhost:4443".to_string()),
+            access_token: creds,
         })
     }
+
+    pub async fn patch_object(
```

Comment:
By passes GCS client because of incompatibility with the fake GCP server that is used for unit tests. It is a work around until we find a better solution.